core/src/main/java/nu/yona/server/subscriptions/service/UserService.java

```diff
@@ -143,7 +143,9 @@ public User addUserCreatedOnBuddyRequest(UserDTO buddyUser, String tempPassword)
 	{
 		UUID savedUserID = CryptoSession.execute(Optional.of(tempPassword), null,
 				() -> tempEncryptionContextExecutor.addUserCreatedOnBuddyRequest(buddyUser).getID());
-		return getEntityByID(savedUserID);
+		User userEntity = getEntityByID(savedUserID);
+		sendMobileNumberConfirmationMessage(userEntity);
```

Comment:
Should we not use a different SMS template here? Because the user may not have read her email yet, the SMS may be quite unexpected.